src/vterm/vterm.h

```diff
@@ -499,18 +499,18 @@ void vterm_state_send_selection(VTermState *state, VTermSelectionMask mask, VTer
 // ------------
 
 typedef struct {
-    unsigned int bold      : 1;
```

Comment:
removing bitfields are going to expand the size of these structs significantly.

If we need to use these in tests it would be better to write parts of the test logic as C helper functions (there is some predecendent for this, marktree e.g. has `ifndef NDEBUG` functions only for debug builds).

Reply:
or if we don't need them we just need to filter out them so luajit doesn't need to see them and get angy..

Reply:
I suspect they're not needed since both the vterm and our own tests pass without them, but I am not entirely sure. I did it this way for now to unblock the work.

Reply:
Did you not read the first sentence?

Reply:
I did but I don't know how to filter them out like he suggested.

Reply:
I did, but I misunderstood. I've opted to filter out all bitfields in unit tests.